### PR TITLE
Demo code for go and php client & server

### PR DESCRIPTION
**Pull request by sokac**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+all:
```
Comment (jmikkola) [inline Makefile:2]: .PHONY: all

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+made from root directory.
+
+# make
+Thrift generated files are already included in this repository (under gen-go and gen-php). If you
+want to rebuild, you can run `make` (the output should be the same if Thrift version is 0.9.3).
+
+# Servers
+
+## PHP
+To start a PHP server, run:
```
Comment (alexkay) [inline README.md:19]: Do we even need PHP servers?

File: Makefile (hunk 1 above)
Comment (sokac) [reply Makefile:2]: done

File: README.md (hunk 2 above)
Comment (sokac) [reply README.md:19]: We don't (at TT). But could be useful for understanding how everything works.

**Comment by alexkay** [comment]
+1

**Pull request merged**